# German translations of dnf-plugins-extras.
# Mario Blättermann <mario.blaettermann@gmail.com>, 2015.
# Mario Blättermann <mario.blaettermann@gmail.com>, 2015. #zanata
# Mario Blättermann <mario.blaettermann@gmail.com>, 2016. #zanata
# Marek Blaha <mblaha@redhat.com>, 2018. #zanata
# Fabian Affolter <mail@fabian-affolter.ch>, 2020.
# Ettore Atalan <atalanttore@googlemail.com>, 2021.
# Joachim Philipp <joachim.philipp@gmail.com>, 2022.
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-09-19 03:35+0000\n"
"PO-Revision-Date: 2022-04-14 20:17+0000\n"
"Last-Translator: Joachim Philipp <joachim.philipp@gmail.com>\n"
"Language-Team: German <https://translate.fedoraproject.org/projects/dnf/dnf-"
"plugins-extras/de/>\n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 4.11.2\n"

#: plugins/kickstart.py:57
msgid "Install packages defined in a kickstart file on your system"
msgstr ""
"In einer Kickstart-Datei auf Ihrem System definierte Pakete installieren"

#: plugins/kickstart.py:62
msgid "kickstart file"
msgstr "*kickstart Datei"

#: plugins/kickstart.py:80
#, python-format
msgid "file cannot be parsed: %s"
msgstr "Datei kann nicht eingelesen werden: %s"

#: plugins/rpm_conf.py:65
msgid "Adding '{}' to list of handling packages for rpmconf"
msgstr "»{}« wird zur Liste der Pakete für rpmconf hinzugefügt"

#: plugins/rpm_conf.py:71
msgid "rpmconf plugin will not run in non-interactive mode"
msgstr ""
"*rpmconf #Plugin wird nicht funktionieren in nicht-interaktives Verfahren"

#: plugins/rpm_conf.py:84
msgid "ignoring sys.exit from rpmconf due to missing MERGE variable"
msgstr ""
"Überhören *sys.Ausgang von *rpmconf dank fehlend #EINVERLEIBEN Variable"

#: plugins/rpm_conf.py:88
msgid "ignoring sys.exit from rpmconf due to missing file"
msgstr "ignoriere sys.exit von rpmconf wegen fehlender Datei"

#: plugins/showvars.py:46
msgid "show all active dnf variables"
msgstr "alle aktiven dnf-Variablen anzeigen"

#: plugins/snapper.py:49
#, python-format
msgid "connect to snapperd failed: %s"
msgstr "Verbindung zu snapperd fehlgeschlagen: %s"

#: plugins/snapper.py:55
msgid "creating pre_snapshot"
msgstr "Erstelle pre_snapshot"

#: plugins/snapper.py:61
#, python-format
msgid "created pre_snapshot %d"
msgstr "pre_snapshot %d erstellt"

#: plugins/snapper.py:65
#, python-format
msgid "creating pre_snapshot failed: %s"
msgstr "Erstellen des pre_snapshot fehlgeschlagen: %s"

#: plugins/snapper.py:74
msgid "skipping post_snapshot because creation of pre_snapshot failed"
msgstr ""
"post_snapshot wird übersprungen, da Erstellen des pre_snapshot "
"fehlgeschlagen ist"

#: plugins/snapper.py:80
msgid "creating post_snapshot"
msgstr "Erstelle post_snapshot"

#: plugins/snapper.py:85
#, python-format
msgid "created post_snapshot %d"
msgstr "post_snapshot %d erstellt"

#: plugins/snapper.py:89
#, python-format
msgid "creating post_snapshot failed: %s"
msgstr "Erstellen des post_snapshot fehlgeschlagen: %s"

#. Translators: This string is only used in unit tests.
#: plugins/system_upgrade.py:45
msgid "the color of the sky"
msgstr "die Farbe des Himmels"

#: plugins/system_upgrade.py:57
msgid "Need a --releasever greater than the current system version."
msgstr ""
"Es wird eine höhere --releasever benötigt als die installierte Version."

#. Translators: do not change "reboot" here
#: plugins/system_upgrade.py:59
#, python-brace-format
msgid ""
"Download complete! Use 'dnf {command} reboot' to start the upgrade.\n"
"To remove cached metadata and transaction use 'dnf {command} clean'"
msgstr ""
"Download abgeschlossen! Verwenden Sie 'dnf {command} reboot', um das Upgrade "
"zu starten.\n"
"Um zwischengespeicherte Metadaten und Transaktionen zu entfernen, verwenden "
"Sie 'dnf {command} clean'"

#: plugins/system_upgrade.py:62
msgid "Sorry, you need to use 'download --releasever' instead of '--network'"
msgstr ""

#: plugins/system_upgrade.py:71
msgid "Reboot turned off, not rebooting."
msgstr ""

#: plugins/system_upgrade.py:122
#, python-format
msgid "Screen blanking can't be disabled: %s"
msgstr ""

#: plugins/system_upgrade.py:142
#, python-format
msgid "Failed loading state file: %s, continuing with empty state."
msgstr ""

#: plugins/system_upgrade.py:288
msgid "The following boots appear to contain upgrade logs:"
msgstr ""

#: plugins/system_upgrade.py:298
msgid "-- no logs were found --"
msgstr "-- Keine Logs gefunden --"

#: plugins/system_upgrade.py:313
msgid "Cannot find logs with this index."
msgstr ""

#: plugins/system_upgrade.py:322
msgid "Unable to match systemd journal entry"
msgstr ""

#: plugins/system_upgrade.py:343
msgid "Prepare system for upgrade to a new release"
msgstr ""

#: plugins/system_upgrade.py:359
msgid "keep installed packages if the new release's version is older"
msgstr ""
"installierte Pakete beibehalten, wenn die Fassung der neuen Version älter ist"

#: plugins/system_upgrade.py:363
msgid "which logs to show"
msgstr "welche Protokolle angezeigt werden sollen"

#: plugins/system_upgrade.py:397
#, python-brace-format
msgid "Incompatible version of data. Rerun 'dnf {command} download [OPTIONS]'"
msgstr ""

#: plugins/system_upgrade.py:434
msgid "Command 'offline-distrosync' cannot be used with --no-downgrade option"
msgstr ""

#: plugins/system_upgrade.py:456
msgid "Additional information for System Upgrade: {}"
msgstr ""

#: plugins/system_upgrade.py:459
msgid ""
"Before you continue ensure that your system is fully upgraded by running "
"\"dnf --refresh upgrade\". Do you want to continue"
msgstr ""
"Bevor Sie fortfahren, stelle Sie bitte sicher, das alle Aktualisierungen "
"installiert sind. Dies kann durch den Aufruf von \"dnf --refresh upgrade\" "
"erfolgen. Wollen Sie fortfahren? "

#: plugins/system_upgrade.py:463
msgid "Operation aborted."
msgstr "Operation abgebrochen."

#: plugins/system_upgrade.py:521
msgid "system is not ready for upgrade"
msgstr ""

#: plugins/system_upgrade.py:524
#, python-brace-format
msgid ""
"the transaction was not prepared for '{command}'. Rerun 'dnf {command} "
"download [OPTIONS]'"
msgstr ""

#: plugins/system_upgrade.py:528
msgid "upgrade is already scheduled"
msgstr ""

#: plugins/system_upgrade.py:534
msgid "trigger file does not exist. exiting quietly."
msgstr ""

#: plugins/system_upgrade.py:537
msgid "another upgrade tool is running. exiting quietly."
msgstr ""

#: plugins/system_upgrade.py:546
#, python-brace-format
msgid "use 'dnf {command} reboot' to begin the upgrade"
msgstr ""

#: plugins/system_upgrade.py:564
msgid "Rebooting to perform upgrade."
msgstr ""

#: plugins/system_upgrade.py:597
msgid "Starting offline upgrade. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:599
msgid "Starting offline distrosync. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:601
msgid "Starting system upgrade. This will take a while."
msgstr ""

#: plugins/system_upgrade.py:617
msgid "Cleaning up downloaded data..."
msgstr "Heruntergeladene Daten werden bereinigt ..."

#: plugins/system_upgrade.py:647
msgid ""
"The system-upgrade transaction is empty, your system is already up-to-date."
msgstr ""

#: plugins/system_upgrade.py:656
msgid "Transaction saved to {}."
msgstr "Transaktion gespeichert in {}."

#: plugins/system_upgrade.py:659
msgid "Error storing transaction: {}"
msgstr "Fehler beim Speichern der Transaktion: {}"

#: plugins/system_upgrade.py:681
msgid "Download finished."
msgstr "Herunterladen abgeschlossen."

#: plugins/system_upgrade.py:684 plugins/system_upgrade.py:685
msgid "Upgrade complete! Cleaning up and rebooting..."
msgstr ""

#: plugins/system_upgrade.py:694
msgid "Prepare offline upgrade of the system"
msgstr ""

#: plugins/system_upgrade.py:699
msgid "Prepare offline distrosync of the system"
msgstr ""

#: plugins/torproxy.py:31
msgid "Disabling torproxy plugin: cannot connect to the Tor network"
msgstr ""

#: plugins/tracer.py:69
msgid "For more information run:"
msgstr "Weitere Informationen erhalten Sie mit:"

#: plugins/tracer.py:96
msgid "Call to Tracer API ended unexpectedly:"
msgstr ""

#: plugins/tracer.py:98
msgid ""
"Please visit https://github.com/FrostyX/tracer/issues and submit the issue. "
"Thank you"
msgstr ""

#: plugins/tracer.py:100
msgid "We apologize for any inconvenience"
msgstr "Wir bitten, die Unannehmlichkeiten zu entschuldigen"
